force-app/main/default/aura/nksChatAuthenticationInfoContainer/nksChatAuthenticationInfoContainerHelper.js

```diff
@@ -6,74 +6,80 @@
-        // Get the empApi component
         const empApi = component.find('empApi');
-        // Get the channel
-        const channel = '/topic/Chat_Auth_Status_Changed';
-        // Replay option to get new events
+        const channel = '/data/MessagingSessionChangeEvent';
         const replayId = -1;
 
-        // Subscribe to an event
         empApi
             .subscribe(
                 channel,
                 replayId,
                 $A.getCallback((eventReceived) => {
-                    this.onEmpApiEvent(component, eventReceived);
+                    this.handleEmpApiEvent(component, eventReceived);
                 })
             )
             .then((subscription) => {
-                // Save subscription to unsubscribe later
                 component.set('v.subscription', subscription);
+                console.log('Subscribed to channel:', channel);
+            })
+            .catch((error) => {
+                console.error('Subscription error:', JSON.stringify(error));
             });
     },
 
-    // Invokes the unsubscribe method on the empApi component
+    // Unsubscribes from the Change Data Capture event for MessagingSession
     unsubscribeEmpApi: function (component) {
-        // Get the empApi component
         const empApi = component.find('empApi');
-        // Get the subscription that we saved when subscribing
         const subscription = component.get('v.subscription');
 
-        // Unsubscribe from event
-        empApi.unsubscribe(
-            subscription,
-            $A.getCallback(() => {
-                // Confirm that we have unsubscribed from the event channel
-                component.set('v.subscription', null);
-            })
-        );
+        if (subscription) {
+            empApi
+                .unsubscribe(
+                    subscription,
+                    $A.getCallback(() => {
+                        component.set('v.subscription', null);
+                        console.log('Unsubscribed from channel');
+                    })
+                )
+                .catch((error) => {
+                    console.error('Unsubscribe error:', JSON.stringify(error));
+                });
+        }
     },
 
-    //When auth status changes update the tab icon
-    onEmpApiEvent: function (component, eventReceived) {
-        const authStatus = eventReceived.data.sobject.CRM_Authentication_Status__c;
-        const changedRecordId = eventReceived.data.sobject.Id;
+    // Handles Change Data Capture event for changes in authentication status
+    handleEmpApiEvent: function (component, eventReceived) {
         const recordId = component.get('v.recordId');
+        const eventRecordId = eventReceived.data.payload.ChangeEventHeader.recordIds[0];
+        const changedFields = eventReceived.data.payload.ChangeEventHeader.changedFields;
+
+        if (eventRecordId === recordId && changedFields.includes('CRM_Authentication_Status__c')) {
```

Comment:
Æ ser i dokumentasjonen at det vanligvis bare e en recordId her, men det står at Salesforce kan merge flere updates inn i en event, så æ trur en enkel loop igjennom alle recordIds for å se om en av de matche e smartar enn å bare ta den første.